examples/problem_decomposition/mifno.ipynb

```diff
@@ -210,14 +214,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from tangelo.problem_decomposition import MIFNOHelper"
+    "from tangelo.problem_decomposition import MethodOfIncrementsHelper"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`MIFNOHelper` can either directly take a \"results\" object in json format produced by QEMIST Cloud (using `mifno_full_result=...`), or the path to a file containing that object (using `mifno_log_file=...`). In our example, let's just assume the full results has been saved in `../data/BeH2_STO3G_MIFNO_trunc2.json` and let's see what information has been retrieved."
+    "`MethodOfIncrementsHelper` can either directly take a \"results\" object in json format produced by QEMIST Cloud (using `full_result=...`), or the path to a file containing that object (using `log_file=...`). In our example, let's just assume the full results has been saved in a folder called `BeH2_CCPVDZ_MIFNO_HBCI` and let's see what information has been retrieved."
```

Comment:
Note: MIFNO is in the file name here. Sfter your intro of both MI-FNO and iFCI i think you should say that this is a MI-FNO job (and if I understood your implementation well, highlight that it works the same way for iFCI).